Modules/Core/Common/include/itkRealTimeStamp.h

```diff
@@ -55,6 +55,13 @@ class ITKCommon_EXPORT RealTimeStamp
   /** Destructor */
   ~RealTimeStamp();
 
+  RealTimeStamp(const RealTimeStamp &) = default;
```

Comment:
The default constructor can be removed too, as [these assignments](https://github.com/InsightSoftwareConsortium/ITK/blob/f408b77a57095a44964187af840353809f3644b0/Modules/Core/Common/src/itkRealTimeStamp.cxx#L71-L72) can be made at the point of declaration. Then we could have rule of zero, right?

Reply:
Not sure, what about this
https://github.com/InsightSoftwareConsortium/ITK/blob/f408b77a57095a44964187af840353809f3644b0/Modules/Core/Common/src/itkRealTimeStamp.cxx#L80?

Reply:
I guess we can't have any constructors if we want "rule of zero". Not a show stopper, but we could default the default constructor too.

Reply:
@dzenanz Indeed. I believe the warnings might also be fixed by just removing the user-defined destructor, `~RealTimeStamp()`. Optionally, the default constructor might indeed be defaulted by `=default`. 

Doing so would not change the API, but it would be a very small _ABI_ change (regarding the binary interface), though. Because it would make those member functions _inline_  (header-only), while they are now still defined in the CXX file. No big deal to me, just something to take into account.

The current version of the PR does not change the API, it does not even change the _ABI_. As far as i can see  😃

Reply:
Thanks for the comments. Leaving rule of zero for a separate PR when a new major ITK version will be considered. Resolving. Ready to be merged.